main/nagios-plugins/nagios-plugins.post-upgrade

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+cat <<EOF
+NOTE: Since 2.0.3-r2 all plugins are separated into subpackages
```

Comment:
Agree, but as you wrote, don't know how to print it conditionally based on previous version.